project/src/components/add-review-form/add-review-form.tsx

```diff
@@ -31,15 +35,31 @@ function AddReviewForm(): JSX.Element {
   const [rating, setRating] = useState(0);
   const [reviewText, setReviewText] = useState('');
 
+  const dispatch = useAppDispatch();
+
   const handleRatingChange = (id: number): void => {
     setRating(id);
   };
 
+  const onSubmit = (reviewData: ReviewData) => {
+    dispatch(pushCommentAction(reviewData));
+  };
```

Comment:
Мне кажется можно не выносить отдельно а сразу делать 
```suggestion
  const onSubmit = (reviewData: ReviewData) => {
    dispatch(pushCommentAction({
      comment: reviewText,
      ...
    }));
  };
```